slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeJsonProvider.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection.interchange;
+
+import static com.fasterxml.jackson.databind.SerializationFeature.FLUSH_AFTER_WRITE_VALUE;
+
+import tech.pegasys.web3signer.common.JacksonSerializers.HexDeserialiser;
+import tech.pegasys.web3signer.common.JacksonSerializers.HexSerialiser;
+import tech.pegasys.web3signer.common.JacksonSerializers.StringUInt64Deserializer;
+import tech.pegasys.web3signer.common.JacksonSerializers.StringUInt64Serialiser;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt64;
+
+public class InterchangeJsonProvider {
+  private final JsonMapper jsonMapper;
+
+  public InterchangeJsonProvider() {
+    jsonMapper =
+        JsonMapper.builder()
+            .configure(FLUSH_AFTER_WRITE_VALUE, true)
+            .enable(SerializationFeature.INDENT_OUTPUT)
+            .build();
+    registerInterchangeModule();
```

Comment:
TO DISCUSS: how are you ensuring the variously configured JsonMappers that this replaces are bug-free?

Reply:
in the main (non-test) code, there was only one instance which was shared across manager/importer/exporter. In the test code, we had multiple instance creation utilizing same `InterchangeModule`. Now all places uses JsonMapper constructed using same settings. (and that the tests pass without any problems).